Sources/Logging/Strings/StoreKitStrings.swift

```diff
@@ -77,6 +79,10 @@ extension StoreKitStrings: CustomStringConvertible {
         case .sk1_discount_missing_locale:
             return "There is an issue with the App Store, this SKProductDiscount is missing a Locale - " +
             "The current device Locale will be used instead."
+
+        case .no_cached_requests_and_products_starting_skproduct_request(let identifiers):
+            return "No existing requests and " +
+                "products not cached, starting SKProducts request for: \(identifiers)"
```

Comment:
Ditto, you're still using this in SK2, maybe change it to `Store product request`?

Reply:
was leaving this one in case we decided to cache requests... might be worth a different string until we decide to cache

Reply:
We cache products, just not concurrent requests, which might be good enough?